schemas.php

```diff
@@ -0,0 +1,167 @@
+<?php
+/**
+ * Schema definitions for all widget types.
+ */
+
+$schemas = array(
```

Comment:
This is temporary. It will be added to `\WP_REST_Widgets_Controller::get_type_schema()` and applied to the given Core widget type supplied, and eventually we'll use the schema that is defined on `WP_Widget` itself.